core/src/main/scala/cats/ErrorControl.scala

```diff
@@ -0,0 +1,38 @@
+package cats
+
+import cats.data.EitherT
+
+trait ErrorControl[F[_], G[_], E] extends Serializable {
+  val monadErrorF: MonadError[F, E]
```

Comment:
I don't like using abstract vals on traits since it invites the val init order bug. Can we make these def?